Adding Windows.System.RemoteSystems to an embedded project causes error 'CS0426: The type name 'Vftbl' does not exist in the type 'T?''

**Describe the bug**
When the Windows.System.RemoteSystems namespace, or a type within it, is added to an embedded project it starts outputting the compiler error; `Nullable.cs(17,36,17,41): error CS0426: The type name 'Vftbl' does not exist in the type 'T?'`

**To Reproduce**
1. Open the sample 'TestEmbedded.sln' solution in VS2019 or CS2022
2. Add namespace reference Windows.System.RemoteSystems; to the CsWinRTIncludes section of TestEmbeddedLibrary.csproj
3. Build the project and see the error is introduced (along with other errors about missing types referenced by RemoteSystems.cs; fixing these errors by importing additional namespaces does not fix the error in Nullable). 

The error appears consistently across all target frameworks in the multi-targeted sample (net6.0-windows target fails to build in VS2019 as expected).

**Expected behavior**
The precise repro case described above will still fail to compile with CS0234 errors in RemoteSystems.cs but the CS0426 error in Nullable.cs should not be present as it does not appear to be end user fixable. 

**Version Info**
CSWinRT NuGet :  1.4.1
Windows SDK  : Reproduced on 18362 and 19041
Visual Studio: 16.11.8, 17.0.4

--- a/src/WinRT.Runtime/Interop/IAgileReference.net5.cs
+++ b/src/WinRT.Runtime/Interop/IAgileReference.net5.cs
@@ -160,7 +160,7 @@ namespace ABI.WinRT.Interop
 
         public static implicit operator IGlobalInterfaceTable(IObjectReference obj) => (obj != null) ? new IGlobalInterfaceTable(obj) : null;
         public static implicit operator IGlobalInterfaceTable(ObjectReference<Vftbl> obj) => (obj != null) ? new IGlobalInterfaceTable(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();

--- a/src/WinRT.Runtime/Interop/IAgileReference.netstandard2.0.cs
+++ b/src/WinRT.Runtime/Interop/IAgileReference.netstandard2.0.cs
@@ -91,7 +91,7 @@ namespace ABI.WinRT.Interop
 
         public static implicit operator IAgileReference(IObjectReference obj) => (obj != null) ? new IAgileReference(obj) : null;
         public static implicit operator IAgileReference(ObjectReference<Vftbl> obj) => (obj != null) ? new IAgileReference(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();
@@ -181,7 +181,7 @@ namespace ABI.WinRT.Interop
 
         public static implicit operator IGlobalInterfaceTable(IObjectReference obj) => (obj != null) ? new IGlobalInterfaceTable(obj) : null;
         public static implicit operator IGlobalInterfaceTable(ObjectReference<Vftbl> obj) => (obj != null) ? new IGlobalInterfaceTable(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();

--- a/src/WinRT.Runtime/Projections/DataErrorsChangedEventArgs.cs
+++ b/src/WinRT.Runtime/Projections/DataErrorsChangedEventArgs.cs
@@ -39,7 +39,7 @@ namespace ABI.Microsoft.UI.Xaml.Data
 
         public static implicit operator WinRTDataErrorsChangedEventArgsRuntimeClassFactory(IObjectReference obj) => (obj != null) ? new WinRTDataErrorsChangedEventArgsRuntimeClassFactory(obj) : null;
         public static implicit operator WinRTDataErrorsChangedEventArgsRuntimeClassFactory(ObjectReference<Vftbl> obj) => (obj != null) ? new WinRTDataErrorsChangedEventArgsRuntimeClassFactory(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();

--- a/src/WinRT.Runtime/Projections/IDictionary.net5.cs
+++ b/src/WinRT.Runtime/Projections/IDictionary.net5.cs
@@ -297,7 +297,12 @@ namespace ABI.System.Collections.Generic
 {
     using ABI.Windows.Foundation.Collections;
 
-    public static class IDictionaryMethods<K, V>
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class IDictionaryMethods<K, V>
     {
         public static int get_Count(IObjectReference obj)
         {

--- a/src/WinRT.Runtime/Projections/IEnumerable.net5.cs
+++ b/src/WinRT.Runtime/Projections/IEnumerable.net5.cs
@@ -118,7 +118,12 @@ namespace ABI.System.Collections.Generic
     using global::System;
     using global::System.Runtime.CompilerServices;
 
-    public static class IEnumerableMethods<T>
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class IEnumerableMethods<T>
     {
         public static global::System.Collections.Generic.IEnumerator<T> GetEnumerator(IObjectReference obj)
         {

--- a/src/WinRT.Runtime/Projections/IEnumerable.net5.cs
+++ b/src/WinRT.Runtime/Projections/IEnumerable.net5.cs
@@ -59,7 +59,7 @@ namespace ABI.Windows.Foundation.Collections
     [Guid("FAA585EA-6214-4217-AFDA-7F46DE5869B3")]
     internal interface IIterable<T> : ABI.System.Collections.Generic.IEnumerable<T>
     {
-        public static Guid PIID = ABI.System.Collections.Generic.IEnumerable<T>.PIID;
+        public static new Guid PIID = ABI.System.Collections.Generic.IEnumerable<T>.PIID;
     }
 }
 
@@ -379,7 +379,7 @@ namespace ABI.System.Collections.Generic
             }
         }
 
-        public sealed class ToAbiHelper : global::Windows.Foundation.Collections.IIterator<T>, IBindableIterator
+        public sealed class ToAbiHelper : global::Windows.Foundation.Collections.IIterator<T>, global::Microsoft.UI.Xaml.Interop.IBindableIterator
         {
             private readonly global::System.Collections.Generic.IEnumerator<T> m_enumerator;
             private bool m_firstItem = true;
@@ -469,7 +469,7 @@ namespace ABI.System.Collections.Generic
 
             public bool MoveNext() => _MoveNext();
 
-            uint IBindableIterator.GetMany(ref object[] items)
+            uint global::Microsoft.UI.Xaml.Interop.IBindableIterator.GetMany(ref object[] items)
             {
                 // Should not be called.
                 throw new NotImplementedException();

--- a/src/WinRT.Runtime/Projections/IEnumerable.netstandard2.0.cs
+++ b/src/WinRT.Runtime/Projections/IEnumerable.netstandard2.0.cs
@@ -342,7 +342,7 @@ namespace ABI.System.Collections.Generic
             }
         }
 
-        public sealed class ToAbiHelper : global::Windows.Foundation.Collections.IIterator<T>, IBindableIterator
+        public sealed class ToAbiHelper : global::Windows.Foundation.Collections.IIterator<T>, global::Microsoft.UI.Xaml.Interop.IBindableIterator
         {
             private readonly global::System.Collections.Generic.IEnumerator<T> m_enumerator;
             private bool m_firstItem = true;
@@ -432,7 +432,7 @@ namespace ABI.System.Collections.Generic
 
             public bool MoveNext() => _MoveNext();
 
-            uint IBindableIterator.GetMany(ref object[] items)
+            uint global::Microsoft.UI.Xaml.Interop.IBindableIterator.GetMany(ref object[] items)
             {
                 // Should not be called.
                 throw new NotImplementedException();

--- a/src/WinRT.Runtime/Projections/IList.net5.cs
+++ b/src/WinRT.Runtime/Projections/IList.net5.cs
@@ -150,7 +150,12 @@ namespace ABI.System.Collections.Generic
     using global::System;
     using global::System.Runtime.CompilerServices;
 
-    public static class IListMethods<T> {
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class IListMethods<T> {
         
         public static int get_Count(IObjectReference obj)
         {

--- a/src/WinRT.Runtime/Projections/IReadOnlyDictionary.net5.cs
+++ b/src/WinRT.Runtime/Projections/IReadOnlyDictionary.net5.cs
@@ -180,7 +180,12 @@ namespace ABI.System.Collections.Generic
     using global::System;
     using global::System.Runtime.CompilerServices;
 
-    public static class IReadOnlyDictionaryMethods<K, V>
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class IReadOnlyDictionaryMethods<K, V>
     {
         public static int get_Count(IObjectReference obj)
         {

--- a/src/WinRT.Runtime/Projections/IReadOnlyList.net5.cs
+++ b/src/WinRT.Runtime/Projections/IReadOnlyList.net5.cs
@@ -166,7 +166,12 @@ namespace ABI.System.Collections.Generic
     using global::System;
     using global::System.Runtime.CompilerServices;
 
-    public static class IReadOnlyListMethods<T>
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class IReadOnlyListMethods<T>
     {
         public static int get_Count(IObjectReference obj)
         {

--- a/src/WinRT.Runtime/Projections/IReferenceArray.net5.cs
+++ b/src/WinRT.Runtime/Projections/IReferenceArray.net5.cs
@@ -121,7 +121,7 @@ namespace ABI.Windows.Foundation
 
         public static implicit operator IReferenceArray<T>(IObjectReference obj) => (obj != null) ? new IReferenceArray<T>(obj) : null;
         public static implicit operator IReferenceArray<T>(ObjectReference<Vftbl> obj) => (obj != null) ? new IReferenceArray<T>(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();

--- a/src/WinRT.Runtime/Projections/IReferenceArray.netstandard2.0.cs
+++ b/src/WinRT.Runtime/Projections/IReferenceArray.netstandard2.0.cs
@@ -125,7 +125,7 @@ namespace ABI.Windows.Foundation
 
         public static implicit operator IReferenceArray<T>(IObjectReference obj) => (obj != null) ? new IReferenceArray<T>(obj) : null;
         public static implicit operator IReferenceArray<T>(ObjectReference<Vftbl> obj) => (obj != null) ? new IReferenceArray<T>(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();

--- a/src/WinRT.Runtime/Projections/NotifyCollectionChangedEventArgs.cs
+++ b/src/WinRT.Runtime/Projections/NotifyCollectionChangedEventArgs.cs
@@ -33,7 +33,7 @@ namespace ABI.Microsoft.UI.Xaml.Interop
         internal static ObjectReference<Vftbl> FromAbi(IntPtr thisPtr) => ObjectReference<Vftbl>.FromAbi(thisPtr);
 
         public static implicit operator INotifyCollectionChangedEventArgs(IObjectReference obj) => (obj != null) ? new INotifyCollectionChangedEventArgs(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IObjectReference ObjRef { get => _obj; }
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
@@ -125,7 +125,7 @@ namespace ABI.Microsoft.UI.Xaml.Interop
 
         public static implicit operator WinRTNotifyCollectionChangedEventArgsRuntimeClassFactory(IObjectReference obj) => (obj != null) ? new WinRTNotifyCollectionChangedEventArgsRuntimeClassFactory(obj) : null;
         public static implicit operator WinRTNotifyCollectionChangedEventArgsRuntimeClassFactory(ObjectReference<Vftbl> obj) => (obj != null) ? new WinRTNotifyCollectionChangedEventArgsRuntimeClassFactory(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();

--- a/src/WinRT.Runtime/Projections/Nullable.cs
+++ b/src/WinRT.Runtime/Projections/Nullable.cs
@@ -14,15 +14,15 @@ namespace ABI.Windows.Foundation
 
     internal static class BoxedValueIReferenceImpl<T>
     {
-        private static Nullable<T>.Vftbl AbiToProjectionVftable;
+        private static global::ABI.System.Nullable<T>.Vftbl AbiToProjectionVftable;
         public static IntPtr AbiToProjectionVftablePtr;
 
         static unsafe BoxedValueIReferenceImpl()
         {
-            AbiToProjectionVftable = new Nullable<T>.Vftbl
+            AbiToProjectionVftable = new global::ABI.System.Nullable<T>.Vftbl
             {
                 IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-                get_Value_0 = global::System.Delegate.CreateDelegate(Nullable<T>.Vftbl.get_Value_0_Type, typeof(BoxedValueIReferenceImpl<T>).GetMethod("Do_Abi_get_Value_0", BindingFlags.NonPublic | BindingFlags.Static).MakeGenericMethod(Marshaler<T>.AbiType))
+                get_Value_0 = global::System.Delegate.CreateDelegate(global::ABI.System.Nullable<T>.Vftbl.get_Value_0_Type, typeof(BoxedValueIReferenceImpl<T>).GetMethod("Do_Abi_get_Value_0", BindingFlags.NonPublic | BindingFlags.Static).MakeGenericMethod(Marshaler<T>.AbiType))
             };
             var nativeVftbl = (IntPtr*)ComWrappersSupport.AllocateVtableMemory(typeof(BoxedValueIReferenceImpl<T>), Marshal.SizeOf<global::WinRT.IInspectable.Vftbl>() + sizeof(IntPtr) * 1);
             Marshal.StructureToPtr(AbiToProjectionVftable.IInspectableVftbl, (IntPtr)nativeVftbl, false);

--- a/src/WinRT.Runtime/Projections/Nullable.cs
+++ b/src/WinRT.Runtime/Projections/Nullable.cs
@@ -186,7 +186,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, int* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -196,7 +196,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, int*, int>)&Do_Abi_get_Value_0
@@ -207,7 +207,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, int* __return_value__)
@@ -263,7 +263,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, IntPtr* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -273,7 +273,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, IntPtr*, int>)&Do_Abi_get_Value_0
@@ -284,7 +284,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, IntPtr* __return_value__)
@@ -341,7 +341,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, byte* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -351,7 +351,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, byte*, int>)&Do_Abi_get_Value_0
@@ -362,7 +362,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, byte* __return_value__)
@@ -418,7 +418,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, sbyte* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -428,7 +428,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, sbyte*, int>)&Do_Abi_get_Value_0
@@ -439,7 +439,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, sbyte* __return_value__)
@@ -495,7 +495,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, short* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -505,7 +505,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, short*, int>)&Do_Abi_get_Value_0
@@ -516,7 +516,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, short* __return_value__)
@@ -572,7 +572,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, ushort* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -582,7 +582,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, ushort*, int>)&Do_Abi_get_Value_0
@@ -593,7 +593,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, ushort* __return_value__)
@@ -649,7 +649,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, uint* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -659,7 +659,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, uint*, int>)&Do_Abi_get_Value_0
@@ -670,7 +670,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, uint* __return_value__)
@@ -726,7 +726,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, long* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -736,7 +736,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, long*, int>)&Do_Abi_get_Value_0
@@ -747,7 +747,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, long* __return_value__)
@@ -803,7 +803,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, ulong* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -813,7 +813,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, ulong*, int>)&Do_Abi_get_Value_0
@@ -824,7 +824,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, ulong* __return_value__)
@@ -880,7 +880,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, float* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -890,7 +890,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, float*, int>)&Do_Abi_get_Value_0
@@ -901,7 +901,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, float* __return_value__)
@@ -957,7 +957,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, double* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -967,7 +967,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, double*, int>)&Do_Abi_get_Value_0
@@ -978,7 +978,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, double* __return_value__)
@@ -1034,7 +1034,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, char* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -1044,7 +1044,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, char*, int>)&Do_Abi_get_Value_0
@@ -1055,7 +1055,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, char* __return_value__)
@@ -1111,7 +1111,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, bool* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -1121,7 +1121,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, bool*, int>)&Do_Abi_get_Value_0
@@ -1132,7 +1132,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, bool* __return_value__)
@@ -1188,7 +1188,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, Guid* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -1198,7 +1198,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, Guid*, int>)&Do_Abi_get_Value_0
@@ -1209,7 +1209,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, Guid* __return_value__)
@@ -1265,7 +1265,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, DateTimeOffset* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -1275,7 +1275,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, DateTimeOffset*, int>)&Do_Abi_get_Value_0
@@ -1286,7 +1286,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, DateTimeOffset* __return_value__)
@@ -1343,7 +1343,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, TimeSpan* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -1353,7 +1353,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, TimeSpan*, int>)&Do_Abi_get_Value_0
@@ -1364,7 +1364,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, TimeSpan* __return_value__)
@@ -1421,7 +1421,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, IntPtr* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -1431,7 +1431,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, IntPtr*, int>)&Do_Abi_get_Value_0
@@ -1442,7 +1442,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, IntPtr* __return_value__)
@@ -1482,7 +1482,7 @@ namespace ABI.System
             private static readonly Vftbl AbiToProjectionVftable;
             public static readonly IntPtr AbiToProjectionVftablePtr;
 
-#if NETSTANDARD2_0
+#if !NET
             private unsafe delegate int GetValueDelegate(IntPtr thisPtr, Type* value);
             private static readonly GetValueDelegate delegateCache;
 #endif
@@ -1492,7 +1492,7 @@ namespace ABI.System
                 AbiToProjectionVftable = new Vftbl
                 {
                     IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
-#if NETSTANDARD2_0
+#if !NET
                     _Get_Value_0 = Marshal.GetFunctionPointerForDelegate(delegateCache = Do_Abi_get_Value_0).ToPointer()
 #else
                     _Get_Value_0 = (delegate* unmanaged<IntPtr, Type*, int>)&Do_Abi_get_Value_0
@@ -1503,7 +1503,7 @@ namespace ABI.System
                 AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
             }
 
-#if !NETSTANDARD2_0
+#if NET
             [UnmanagedCallersOnly]
 #endif
             private static unsafe int Do_Abi_get_Value_0(IntPtr thisPtr, Type* __return_value__)

--- a/src/WinRT.Runtime/Projections/PropertyChangedEventArgs.cs
+++ b/src/WinRT.Runtime/Projections/PropertyChangedEventArgs.cs
@@ -35,7 +35,7 @@ namespace ABI.Microsoft.UI.Xaml.Data
 
         public static implicit operator WinRTPropertyChangedEventArgsRuntimeClassFactory(IObjectReference obj) => (obj != null) ? new WinRTPropertyChangedEventArgsRuntimeClassFactory(obj) : null;
         public static implicit operator WinRTPropertyChangedEventArgsRuntimeClassFactory(ObjectReference<Vftbl> obj) => (obj != null) ? new WinRTPropertyChangedEventArgsRuntimeClassFactory(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();

--- a/src/WinRT.Runtime/Projections/Uri.cs
+++ b/src/WinRT.Runtime/Projections/Uri.cs
@@ -54,7 +54,7 @@ namespace ABI.System
 
         public static implicit operator WinRTUriRuntimeClassFactory(IObjectReference obj) => (obj != null) ? new WinRTUriRuntimeClassFactory(obj) : null;
         public static implicit operator WinRTUriRuntimeClassFactory(ObjectReference<Vftbl> obj) => (obj != null) ? new WinRTUriRuntimeClassFactory(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();
         public A As<A>() => _obj.AsType<A>();

--- a/src/cswinrt/strings/additions/Windows.Storage.Streams/IMarshal.cs
+++ b/src/cswinrt/strings/additions/Windows.Storage.Streams/IMarshal.cs
@@ -196,7 +196,7 @@ namespace ABI.Com
         internal static ObjectReference<Vftbl> FromAbi(IntPtr thisPtr) => ObjectReference<Vftbl>.FromAbi(thisPtr);
 
         public static implicit operator IMarshal(IObjectReference obj) => (obj != null) ? new IMarshal(obj) : null;
-        protected readonly ObjectReference<Vftbl> _obj;
+        private readonly ObjectReference<Vftbl> _obj;
         public IObjectReference ObjRef { get => _obj; }
         public IntPtr ThisPtr => _obj.ThisPtr;
         public ObjectReference<I> AsInterface<I>() => _obj.As<I>();

